src/cli_input.rs

```diff
@@ -94,7 +97,7 @@ impl CliInput {
                     amount = amount[..amount.len() - 1].to_string();
                 }
 
-                match amount.parse::<f64>() {
+                match CliInput::strip_thousand_separators(&amount).parse::<f64>() {
```

Comment:
```suggestion
                Self::strip_thousand_separators(&amount).parse::<f64>() {
```

Makes code more flexible for later renaming of the struct